test/e2e/resources/table_billing.yaml

```diff
@@ -0,0 +1,19 @@
+# Table used to test multiple interfering updates at once
```

Comment:
I'm not a fan of these file naming conventions. The test files are related to arbitrary attribute definition names, rather than the functionality they're testing. `table_billing` could become `table_basic`, `table_forums` becomes `table_local_secondary_indexes` and `table_offices` becomes `table_global_secondary_indexes`

Reply:
Agreed :+1: